common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -150,6 +150,7 @@ is_giant_culture_trigger = {
 		culture_group = culture_group:ancient_group
 		culture_group = culture_group:shathyar_group
 		culture_group = culture_group:magnataur_group
+		culture_group = culture_group:ogre_group
```

Comment:
Ogres should go in `is_large_culture_trigger` (and they are).
What you wanted to achieve here?

Reply:
This was my mistake 😾